gramps/gui/filters/sidebar/_sidebarfilter.py

```diff
@@ -316,3 +363,47 @@ def set_filters_to_name(self, filter_name):
                 self.generic.set_active_iter(iter)
                 break
             iter = liststore.iter_next(iter)
+
+    def filter_is_ok(self):
+        the_filter = self.get_filter()
+        if the_filter is None:
+            return True
+        for rule in the_filter.get_rules():
+            if isinstance(rule, gramps.gen.filters.rules.place.WithinArea):
+                if rule.list[0] is None: # No active place
+                    msg = _("You should select a place when using the 'Within' rule")
+                    self.msg_label.set_markup("<span color='red'>" + msg + "</span>")
```

Comment:
It is better to add the css error class to the widget style context rather than hard-coding the colour "red".   For example you could use:
```
self.msg_label.set_text(msg)
self.msg_label.get_style_context().add_class("error")
```
This will use the error colour defined in the theme.  This is useful for visually impaired users and non-European cultures.

Reply:
I don't quite understand how css and themes work in Gramps. If I add the "error" class then the label still doesn't change color.

Reply:
Interesting.  I've pushed a commit to your branch which works for me.  The errors are red, but the elapsed time is black.

Reply:
I can make the css style work if I manually make changes in the file ~/.config/gtk-3.0/gtk.css. But that is probably not the correct the way to do this, is it?

Reply:
Anyway, it seems that we are doing changes simultaneously. Did I mess things up?

Reply:
What operating system and theme are you using?  No changes to the css should be necessary.

When you run Gramps in the _master_ branch, what colour is the version number in the status bar?  That is set using the "destructive-action" class.

You can either use my changes or update the code yourself.